acme/acme/client_test.py

```diff
@@ -84,6 +84,127 @@ def setUp(self):
         # Reason code for revocation
         self.rsn = 1
 
+
+class BackwardsCompatibleClientV2Test(ClientTestBase):
+    """Tests for  acme.client.BackwardsCompatibleClientV2."""
+
+    def setUp(self):
```

Comment:
nit: If all `setUp` is doing is calling `setUp` for the parent class, you can omit it entirely.

Reply:
refactoring is fun